instrumentation/google.golang.org/grpc/otelgrpc/interceptorinfo.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelgrpc // import "go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
+
+import (
+	"google.golang.org/grpc"
+)
+
+// interceptorType is the flag to define which gRPC interceptor
+// the interceptorInfo object is.
+type InterceptorType uint8
+
+const (
```

Comment:
Should we introduce a UndefinedClient type as the default?  Seems like generally a good practice since this is exposed to users so it doesn't default to UnaryClient.

Reply:
Agreed. Because `InterceptorType` has server types as well, so how about `UndefinedInterceptor` as default?

Reply:
Yes, that makes more sense